dev_requirements.txt

```diff
@@ -1,11 +1,11 @@
 Django==1.11.15
--e git://github.com/divio/django-cms.git@release/4.0.x#egg=django_cms
+-e git+git@github.com:divio/django-cms.git@release/4.0.x#egg=django_cms
 -e git+git@github.com:FidelityInternational/djangocms-fil-admin-style.git@master#egg=djangocms_fil_admin_style
 -e git+git@github.com:FidelityInternational/djangocms-fil-bootstrap.git@master#egg=djangocms_fil_bootstrap
 -e git+git@github.com:FidelityInternational/djangocms-internalsearch.git@master#egg=djangocms_internalsearch
--e git://github.com/divio/djangocms-versioning.git@master#egg=djangocms_versioning
--e git://github.com/FidelityInternational/djangocms-version-locking.git@master#egg=djangocms_version_locking
+-e git+git@github.com:FidelityInternational/djangocms-versioning.git@master#egg=djangocms_versioning
```

Comment:
@Aiky30 versioning should be pointing to divio's master branch?

Reply:
@webbyfox As it's the dev requirements it should be pointing to the project where you can create a branch and then generate a PR against it. All of the repositories point at the FIL repositories, that is so we can just start making changes.